docs/Changelog.md

```diff
@@ -9633,6 +9633,56 @@ This version of the operator has been available since version 10 of the default
 <dd>Constrain index tensor to int64</dd>
 </dl>
 
+### <a name="Mod-10"></a>**Mod-10**</a>
+
+  Performs element-wise binary modulus (with Numpy-style broadcasting support). 
+      The sign of the remainder is the same as that of the Divisor.
+    
+      Mod operator can also behave like C fmod() or numpy.fmod. In this case, the sign of the remainder however, will be the same as the Dividend 
+      (in contrast to integer mod). To force a behavior like numpy.fmod() an 'fmod' Attribute is provided.
+      This attribute is set to 0 by default causing the behavior to be like integer mod. 
+      Setting this attribute to 1 causes the remainder to be calculated similar to that of numpy.fmod().
+  
+      If the input type is floating point, then `fmod` attribute must be set to 1.
+    
+      In case of dividend being zero, the results will be platform dependent.
```

Comment:
IMHO, standards exists to make sure there is not a platform dependent behavior.

Reply:
@yuslepukhin : Generally agreed. Though, even standards don't full specify degenerate situations (e.g. I can point out many lurky places in the OpenType spec), and this case is at least consistently with the corresponding `Div` operator in being unspecified. If we were to mandate a specific result for all CPU/GPU architectures, then it should be consistent between `Div` and `Mod`.

Reply:
It is difficult to mandate that for various HW backend.

Reply:
This is a good topic for general "error handling" policy for (all?) operators. Currently only a select number of operators allow the user to specify what should be done -- typically in the form of defaults. We need a much more widespread usage of ErrorHandling (or Default) attributes to achieve this form of consistency

Reply:
@yuslepukhin Let me know if you want me to re-open this. thanks